src/warplib/implementations/conversions/fixedBytes.ts

```diff
@@ -67,6 +66,36 @@ export function functionaliseFixedBytesConversion(conversion: FunctionCall, ast:
     ast.replaceNode(conversion, arg);
     return;
   } else {
-    throw new NotSupportedYetError('Narrowing fixed bytes conversion not implemented yet');
+    const fullName = `warp_bytes_narrow${fromType.size === 32 ? '_256' : ''}`;
+    const stub = createCairoFunctionStub(
+      fullName,
+      [
+        ['op', typeNameFromTypeNode(fromType, ast)],
+        ['widthDiff', createUint8TypeName(ast)],
+      ],
+      [['res', typeNameFromTypeNode(toType, ast)]],
+      fromType.size === 32 ? ['range_check_ptr'] : ['bitwise_ptr'],
+      ast,
+      conversion,
+    );
+
+    const call = new FunctionCall(
```

Comment:
you can use this utility here
https://github.com/NethermindEth/warp/blob/warp-ts/src/utils/functionGeneration.ts#L29

Reply:
done